### ISSUE TITLE
Add wrappers for using tsx instead of ts-node

### ISSUE THREAD

**Issue by segevfiner** [typeorm/typeorm#10661]
### Feature Description

Add wrapper scripts for using tsx instead of ts-node. This tends to work better than ts-node.

### The Solution

Add wrapper scripts for using tsx instead of ts-node.

### Considered Alternatives

Wrap the typeorm.js script from node_modules using tsx by yourself 

### Relevant Database Driver(s)

- [ ] aurora-mysql
- [ ] aurora-postgres
- [ ] better-sqlite3
- [ ] cockroachdb
- [ ] cordova
- [ ] expo
- [ ] mongodb
- [ ] mysql
- [ ] nativescript
- [ ] oracle
- [ ] postgres
- [ ] react-native
- [ ] sap
- [ ] spanner
- [ ] sqlite
- [ ] sqlite-abstract
- [ ] sqljs
- [ ] sqlserver

### Are you willing to resolve this issue by submitting a Pull Request?

Yes, I have the time, and I know how to start.

**Comment by predragnikolic** [comment]
why was this closed? :)

**Comment by segevfiner** [comment]
tsx/esbuild doesn't support decorators which are used extensivley by TypeORM

**Comment by predragnikolic** [comment]
I use `tsx` during development with no issues:

```
 "dev": "tsx watch ./src/index.ts",
 "build": "tsc && tsc-alias",
```

I used ts-node before, but tsx is way faster(supports path aliases out of the box) and I do not have to configure anything.

I stumbled upon this https://github.com/evanw/esbuild/issues/104 issue that requests decorators support.
I know that TypeOrm is using experimental decorators a
nd not the actual decorators implemented in TS 5.0.

I wonder how `tsx` works for me during development without any issues.
